packages/rs-drive-abci/src/execution/platform_events/block_fee_processing/process_block_fees/v0/mod.rs

```diff
@@ -243,8 +244,9 @@ mod tests {
             let current_epoch = Epoch::new(epoch_index).unwrap();
             let platform_version = PlatformVersion::latest();
 
-            let block_time_ms =
-                genesis_time_ms + epoch_index as u64 * EPOCH_CHANGE_TIME_MS_V0 + block_height;
+            let block_time_ms = genesis_time_ms
+                + epoch_index as u64 * platform.config.execution.epoch_time_length_s * 1000
```

Comment:
Magic number, let's use constant

Reply:
What are you talking about? it's second to ms. I'm pretty sure 1000 is okay to use.